Project.toml

```
@@ -1,7 +1,7 @@
 name = "Distributions"
 uuid = "31c24e10-a181-5473-b8eb-7969acd0382f"
 authors = ["JuliaStats"]
-version = "0.25.118"
+version = "0.25.119"
```

The PR doesn't change the package, pnly the tests, so we don't need a new release:
```suggestion
version = "0.25.118"
```

---

I made this because `add ForwardDiff@1` leads to "ERROR: Unsatisfiable requirements detected for package Distributions [31c24e10]:", which involved LsqFit... but you may be right that it's not what's bounding ForwardDiff, not really sure.

---

Did you encounter it when trying to update a downstream package?